client/src/components/InventoryView.js

```diff
@@ -27,12 +27,29 @@ class InventoryView extends React.Component {
           this.state.isMobile ? (
             <p className="text-danger">Replace me with a mobile component</p>
           ) : (
-            <InventoryTable />
+            <InventoryTable table={[
```

Comment:
Replace `table` with `inventoryItems`. Aim to be as descriptive as possible when naming your state/props in order to avoid confusion or ambiguity